src/ActionBar/ActionButton.tsx

```diff
@@ -20,7 +20,7 @@ export enum ActionButtonIcons {
 interface ContainerProps {
     disabled?: boolean;
 }
-const Container = styled('div')<ContainerProps>`
+const Container = styled('button')<ContainerProps>`
```

Comment:
Let's force the `type=button` because if the `Container` is added to the form, it will behave as a form submission. From the docs:

> submit: The button submits the form data to the server. This is the default if the attribute is not specified for buttons associated with a `<form>`, or if the attribute is an empty or invalid value.

```suggestion
const Container = styled('button').attr(() => ({type: 'button'}))<ContainerProps>`
```

Reply:
So keep the div and add the styled div with the type button? Because the way you suggested it's a styled button with type button.